runtime/src/iree-amd-aie/aie_runtime/iree_aie_router.cc

```diff
@@ -866,9 +868,19 @@ FailureOr<std::tuple<MasterSetsT, SlaveAMSelsT>> emitPacketRoutingConfiguration(
   for (const auto &[physPortAndID, packetFlowports] : sortedPacketFlows) {
     // The Source Tile of the flow
     TileLoc tileLoc = physPortAndID.physPort.tileLoc;
+    // Make sure the generator for the tile is initialized with the correct
+    // number of arbiters and msels.
+    uint8_t numArbiters =
+        1 + deviceModel.getStreamSwitchArbiterMax(tileLoc.col, tileLoc.row);
```

Comment:
why `+1`?

Reply:
Because `getStreamSwitchArbiterMax` returns the max value, and the first arbiter is `0`, so the number of arbiters is max + 1. This is to be consistent with how it's internally defined in `aie-rt`, and it's consistent with `std::numeric_limits::max` as well.

Reply:
oh duh - whoops sorry poor reading comprehension (or if I blame someone else 😄, `getStreamSwitchArbiterMax`  should be `getStreamSwitchArbiterMaxIdx`)